style.css

```diff
@@ -720,6 +833,71 @@ media query for mobile from 450px to 767px screen
 
   /* work section media query for desktop version */
 
+  /* popup window style start */
+
+  .popUpWinFirst {
+    height: 80%;
+    width: 90%;
+    left: 50px;
+    overflow: hidden;
+  }
+
+  .head h2 {
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+    color: #172b4d;
+  }
+
+  .technologiesPreview {
+    gap: 6px;
+    margin: 12px 0 24px 0;
+  }
+
+  .projectImg {
+    width: 70%;
+    margin: 0;
+  }
+
+  .projectPreview {
+    height: 100%;
+    justify-content: space-between;
+  }
+
+  .project_elements {
+    flex-direction: row;
+    width: 100%;
+    height: 100%;
+    gap: 24px;
+  }
+
+  .details_container {
+    overflow: hidden;
+    width: 40%;
+    height: 78%;
+  }
+
+  .projectDescription {
+    width: 325px;
```

Comment:
- [ ] **[OPTIONAL]** I think it would be better to match the Figma design by adding a max-width property to prevent prevents the [used value](https://developer.mozilla.org/en-US/docs/Web/CSS/used_value) of the [width](https://developer.mozilla.org/en-US/docs/Web/CSS/width) property from becoming larger than the value specified by ```max-width``` 😎😎. 
   - Kindly make sure that you replicate the spacing between your elements to that of the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=IAAiHJLLRY3Kd5Xg-0) 🙏, you can achieve this by giving the picture 60% of the width and the description 40% to match the given design, 
   
   **I have highlighted some spacing issues to be fixed for your reference 👇 👇** 
![image](https://user-images.githubusercontent.com/102757126/218007809-040b014b-6f32-4d0d-b41f-845c52420711.png)